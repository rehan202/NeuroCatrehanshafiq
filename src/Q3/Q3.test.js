import React from 'react';
import ReactDOM from 'react-dom';
import Textbox from './q3';
import { render, cleanup } from '@testing-library/react';

afterEach(cleanup);

it("renders without Crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Textbox></Textbox>, div);

})

it("renders Textbox snapshot test", () => {
    const { asFragment } = render(<Textbox />);
    expect(asFragment()).toMatchSnapshot();
});
