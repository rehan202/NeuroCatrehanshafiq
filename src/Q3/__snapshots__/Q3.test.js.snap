// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Textbox snapshot test 1`] = `
<DocumentFragment>
  <div>
    <input
      data-testid="myTextbox"
      type="text"
      value=""
    />
  </div>
</DocumentFragment>
`;
